### PR TITLE
Some small man page fixes

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: man/systemd.unit.xml [hunk 1]
```diff
@@ -651,10 +651,11 @@
...
 
-        <listitem><para>Takes a boolean argument. If
-        <option>true</option>, this unit will not be stopped when
-        isolating another unit. Defaults to
-        <option>false</option>.</para></listitem>
+        <listitem><para>Takes a boolean argument. If <option>true</option>, this unit
+        will not be stopped when isolating another unit. Defaults to
+        <option>true</option> for service, target, socket, busname, timer, and path
+        units, and <option>false</option> for slice, scope, device, swap, mount, and
+        automount units.</para></listitem>
```
Comment (martinpitt) [inline man/systemd.unit.xml]: The default is the other way around ☺

File: man/systemd.special.xml [hunk 2]
```diff
@@ -328,10 +328,11 @@
...
         <listitem>
-          <para>systemd starts this target whenever Alt+ArrowUp is
-          pressed on the console. This is a good candidate to be
-          aliased (symlinked) to
-          <filename>rescue.target</filename>.</para>
+          <para>systemd starts this target whenever Alt+ArrowUp is pressed on the
+          console. It may be useful to alias this unit to
+          <filename>debug-shell.service</filename> as a last ditch debugging
+          technique. (Note that this is for <emphasis>debugging</emphasis>, and
+          leaves the machine vulnerable to local attackers).</para>
```
Comment (martinpitt) [inline man/systemd.special.xml]: I think the language here should be stronger, or it shouldn't be documented this way at all. For debugging a reproducible hang one can just boot with `systemd.debug-shell`, and debugging a non-reproducible hang would just mean to leave this on permanently - this is quite a gaping local root hole.

File: man/systemd.unit.xml (hunk 1 above)
Comment (keszybz) [reply man/systemd.unit.xml]: Pfff, those negated-by-default options.

File: man/systemd.special.xml (hunk 2 above)
Comment (keszybz) [reply man/systemd.special.xml]: Indeed. I'll add the idea to https://freedesktop.org/wiki/Software/systemd/Debugging/. I changed the text to just give a generic warning without suggesting anything to alias to. Once we have a good emergency shell with authentication, we can revisit this.

**Comment by keszybz** [comment]
https://secure.freedesktop.org/cgit/www/commit/Software/systemd/Debugging.mdwn?id=e8c76e010e04904eec991c3716275e581e645003

**Pull request merged**